scripts/test-summary.sh

```
@@ -0,0 +1,54 @@
+# Count test files
+unit_tests=$(find . -name "*_test.go" -not -name "*_integration_test.go" -not -path "./vendor/*" -not -path "./web/*" | wc -l | tr -d ' ')
+integration_tests=$(find . -name "*_integration_test.go" -not -path "./vendor/*" -not -path "./web/*" | wc -l | tr -d ' ')
+
+echo "Test Files:"
+echo "  Unit test files:        $unit_tests"
+echo "  Integration test files: $integration_tests"
+echo ""
+
+# Count test functions
+echo "Test Functions:"
+unit_funcs=$(grep -h "^func Test" $(find . -name "*_test.go" -not -name "*_integration_test.go" -not -path "./vendor/*" -not -path "./web/*" 2>/dev/null) 2>/dev/null | wc -l | tr -d ' ')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)